.github/pull_request_template.md

```diff
@@ -0,0 +1,81 @@
+# Pull Request - Compound lens model 
```

Comment:
it looks like you modified/added not just the pull request template, but also the description of your specific pull request